include/zephyr/bluetooth/conn.h

```diff
@@ -530,6 +579,112 @@ int bt_conn_get_remote_info(struct bt_conn *conn,
 int bt_conn_le_get_tx_power_level(struct bt_conn *conn,
 				  struct bt_conn_le_tx_power *tx_power_level);
 
+/** @brief Get local enhanced connection transmit power level.
+ *
+ *  @param conn           Connection object.
+ *  @param tx_power       Transmit power level descriptor.
+ *
+ *  @return Zero on success or (negative) error code on failure.
+ *  @retval -ENOBUFS HCI command buffer is not available.
+ */
+int bt_conn_le_enhanced_get_tx_power_level(struct bt_conn *conn,
+					   struct bt_conn_le_tx_power *tx_power);
+
+/** @brief Get remote (peer) transmit power level.
+ *
+ *  @param conn           Connection object.
+ *  @param phy            PHY information.
+ *
+ *  @return Zero on success or (negative) error code on failure.
+ *  @retval -ENOBUFS HCI command buffer is not available.
+ */
+int bt_conn_le_get_remote_tx_power_level(struct bt_conn *conn,
+					 enum bt_conn_le_tx_power_phy phy);
+
+/** @brief Enable transmit power reporting.
+ *
+ *  @param conn           Connection object.
+ *  @param local_enable   Enable/disable reporting for local.
+ *  @param remote_enable  Enable/disable reporting for remote.
+ *
+ *  @return Zero on success or (negative) error code on failure.
+ *  @retval -ENOBUFS HCI command buffer is not available.
+ */
+int bt_conn_le_set_tx_power_report_enable(struct bt_conn *conn,
+					  bool local_enable,
+					  bool remote_enable);
+
+/** @brief Set remote (peer) transmit power.
+ *
+ *  @param conn			Connection object.
+ *  @param phy			PHY bit number i.e. [1M, 2M, s8, s2] == [1, 2, 3, 4]
```

Comment:
```suggestion
 *  @param conn         Connection object.
 *  @param phy          PHY bit number i.e. [1M, 2M, s8, s2] == [1, 2, 3, 4].
```
Replaced tabs with spaces, and aligned with the next line.

Reply:
It looks unaligned here, but is aligned opening the file in VS code. Adding spaces instead of tabs, will see if I have to update again to align then.

Reply:
Tab length is configurable in editors, and many editors truncate the first tab so anything after it aligns, so when some entries have a tabs and others have spaces, the alignment will match for some users and be off for others.